jest.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  roots: ['<rootDir>/src'],
+  transform: {
+    '^.+\\.jsx?$': 'babel-jest',
+    '^.+\\.tsx?$': 'babel-jest',
+  },
+  testURL: 'http://localhost/',
+  setupTestFrameworkScriptFile: 'jest-enzyme',
```

Comment:
This is typically a pointer to a physical file where additional setup can take place. See https://github.com/artsy/force/blob/master/test.config.js